src/test/java/io/cdap/plugin/batch/ETLFTPTestRun.java

```diff
@@ -193,7 +193,7 @@ private void testHelper(Map<String, String> properties, Map<String, String> runT
     DataSetManager<Table> outputManager = getDataset(outputDatasetName);
     List<StructuredRecord> output = MockSink.readOutput(outputManager);
 
-    Assert.assertEquals("Expected records", 1, output.size());
+    Assert.assertEquals("Expected records", 2, output.size());
```

Comment:
I didn't see any test errors when running `mvn clean package`, though now I realize it's because this test run for whatever reason.

Could you also change the pom https://github.com/data-integrations/ftp-plugins/blob/168a3f592fe95e7b3e2c676f1f7e2e0dca4cdbc3/pom.xml#L397 to remove *TestRun.java from the excludes and put it in the includes?

When I do that I'm also seeing guice injection errors. Are you able to successfully run this test?

Reply:
hm I see. Not sure about the history of this project and why it had this failing test class. Could you add `@Ignore` annotations to whatever tests are failing? Will log a bug to investigate some more.

Reply:
Should be done now, `mvn clean package` works locally for me now :)

Reply:
Assuming rebasing on develop HEAD